src/ildasm/windasm.cpp

```diff
@@ -383,6 +383,9 @@ int ProcessOneArg(__in __nullterminated char* szArg, __out char** ppszObjFileNam
         }
         else if ((_stricmp(szOpt, "met") == 0)&&g_fTDC)
         {
+#ifdef FEATURE_CORECLR
+            printf("Warning: 'METADATA' option is ignored for ildasm on CoreCLR.\n");
```

Comment:
We only use stderr for error case where we stop ilasming/ildasming.  I think  so this warning is okay with stdout as it is.